src/Illuminate/Console/Jobs/DispatchCommand.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace Illuminate\Console\Jobs;
+
+use Illuminate\Console\Command;
+use Illuminate\Foundation\Bus\DispatchesJobs;
+
+class DispatchCommand extends Command
+{
+    use DispatchesJobs;
+
+    /**
+     * The console command name.
+     *
+     * @var string
+     */
+    protected $name = 'job:dispatch';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Dispatches a job.';
+
+    /**
+     * @var string The name and signature of the console command.
+     */
+    protected $signature = 'job:dispatch {job : The name of the job that should be dispatched}';
+
+    /**
+     * Execute the console command.
+     */
+    public function fire()
+    {
+        $job = $this->argument('job');
+
+        $this->info("Dispatching $job now...");
```

Comment:
Please use braces around the variable. Although it's not necessary, Laravel normally does it for clarity.